3rdparty/ngraph_bridge/src/ngraph_graph.h

```diff
@@ -187,7 +187,7 @@ inline std::shared_ptr<ngraph::runtime::Backend> GetBackendFromContext(
 
 class OutputElement;
 
-using MapEntry = std::pair<nnvmNodePtr, size_t>;
+using MapEntry = std::pair<const nnvm::Node *, size_t>;
```

Comment:
This should go back to shared_ptr, we're only using it with shared_ptr.get() now